Program.cs

```diff
@@ -1,7 +1,18 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number: ");
+         int n = int.Parse (Console.ReadLine ());
+         int i = 1, count = 1;
+         while (n >= i) {

```

Comment:
The iteration count could be reduced further.

Reply:
Iteration reduced